out/production/hlapp/.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
```

Comment:
Full package `out\production\hlapp` should be removed from repo. It's a compilation result